docs/tutorial/devtools-extension.md

```diff
@@ -1,63 +1,73 @@
 # DevTools Extension
 
-Electron supports the [Chrome DevTools Extension][devtools-extension], which can
-be used to extend the ability of devtools for debugging popular web frameworks.
+Electron supports [Chrome DevTools extensions][devtools-extension], which can
+be used to extend the ability of Chrome's developer tools for debugging
+popular web frameworks.
 
-## How to load a DevTools Extension
+## Manually loading a DevTools extension
 
-This document outlines the process for manually loading an extension.
-You may also try
-[electron-devtools-installer](https://github.com/GPMDP/electron-devtools-installer),
-a third-party tool that downloads extensions directly from the Chrome WebStore.
+To load an extension in Electron, you need to download it via Chrome,
+locate its filesystem path, and then load it into your [Session][session] by calling the
+[`ses.loadExtension`] API.
```

Comment:
It's possible to fetch the .crx with curl and then extract it, which is how I've done it. e.g. https://stackoverflow.com/a/14099762.

Though perhaps it's better to stick with recommending Chrome to download extensions, since the direct request method isn't documented/supported by google.

Reply:
>Though perhaps it's better to stick with recommending Chrome to download extensions, since the direct request method isn't documented/supported by google.
I'd lean towards this reasoning here!